.devcontainer/devcontainer.json

```diff
@@ -39,8 +39,10 @@
            "redhat.vscode-yaml",
            "bierner.markdown-mermaid",
            "d-biehl.robotcode",
-           "bianxianyang.htmlplay"
-            ]
+           "bianxianyang.htmlplay",
+           "DavidAnson.vscode-markdownlint",
+           "EditorConfig.EditorConfig"
```

Comment:
I'm still wondering about this plugin (editor config). We already have some workspace specific VS Code settings. Where do you see the benefit of not specifying the settings there instead compared to having the dedicated .editorconfig file?

Reply:
It would be better if we have only one place for that settings, otherwise we'll get confused what is set and what not. 
We discusses it internally with the others on the team and since editorconfig can also be used for other editors we agreed to have all related setting in .editorconfig file. So the only thing we need to to remove the "files.insertFinalNewline": true from the .vscode/settings.json.
Can you please remove it from there?

Reply:
I also dislike various configuration places. And it's a shame that the eco system did not continue to evolve with that regard... -- It is even very hard to maintain IDE freedom of choice... - Fortunately, there is a plugin for VS.Code to have IntelliJ keyboard shortcuts (even there are many features missing in VS.Code).

I removed the line. And then I learned that `devcontainer.json`'s settings also include the `.vscode/settings.json` things? Otherwise, it would not make sense that the setting only appeared in  `.vscode/settings.json`.

Reply:
Well the `.vscode/settings.json` file is also available inside the workspace mounter in the devcontainer so VS Code is taking them from there as "workspace" settings.